oauthproxy.go

```diff
@@ -605,6 +605,7 @@ func (p *OAuthProxy) SignIn(rw http.ResponseWriter, req *http.Request) {
 		if p.SkipProviderButton {
 			p.OAuthStart(rw, req)
 		} else {
+			// TODO - pass on /oauth2/sign_in query params to /oauth2/start
```

Comment:
Do we want to do this within this PR?

Reply:
I'm not sure - the TODO comment is about what happens if someone linked to `/oauth2/sign_in?prompt=consent` or whatever rather than going straight to `/oauth2/start`, and the provider button is not skipped.  Do we _want_ to pass the parameters on to `/oauth2/start` in that case?

Reply:
Ack that makes sense, happy to have this as a follow up if we aren't ready to resolve this now